src/app/modules/activities-management/components/create-activity/create-activity.component.spec.ts

```diff
@@ -20,6 +21,7 @@ describe('CreateActivityComponent', () => {
   let activityIdtoEditMock;
   let allActivitiesMock;
   let getActivityByIdMock;
+  let getActivityByIdSelectorMock;
```

Comment:
yo can use the variable getActivityByIdMock that is to mock that selector, you don't need to create getActivityByIdSelectorMock